src/Button/styles.ts

```diff
@@ -103,15 +102,15 @@ export const getVariantStyles = (variant: VariantType = 'default', theme?: Theme
       },
     },
     invisible: {
-      color: 'accent.fg',
+      color: 'btn.text',
```

Comment:
@joshblack can you give me an example of what you're thinking?

Reply:
@langermank I probably should have used `btn.hoverBg` or `btn.selectedBg` as a better example 😅 It seemed like if someone created a custom theme like:

```js
const customTheme = deepmerge(theme, {
  colors: {
    btn: {
      hoverBg: 'var(--my-custom-color)',
    },
  },
})
```

Based on the example here: https://primer.style/react/theming#customizing-the-theme

That this might no longer work since that `btn.hoverBg` token is no longer used in the hover styles. Hope that makes sense, let me know if I'm misunderstanding anything!

Reply:
@joshblack I didn't know we offered theming options like that! That's kind of risky isn't it, and makes it near impossible for us to even rename things without any visual changes 😅 but yeah, you're right, it would break. Does that mean I need to move this to a major release?

Reply:
@langermank agreed, I think it brings in component tokens as part of the public API. Would an interim solution potentially be to set the value of these tokens to the new expected value? Not sure if the references work that way but wanted to throw it out there if so.

Also added this to that versioning doc under "theming": https://github.com/primer/react/blob/37cfd07fb1eef4c0655157a0c9025cec94abaed5/contributor-docs/versioning.md#changes so we can align on if this is part of semver or not for the project

Reply:
> Would an interim solution potentially be to set the value of these tokens to the new expected value?

Normally I would say yes, but this component was using generic color tokens here– not specific to Button. I think it would be weird to override `accent.fg` with a different color 🤔 unless I'm not understanding correctly

Reply:
@langermank was more-so thinking of stuff like:

- `btn.hoverBg` has the value `actionListItem.default.hoverBg`
- `btn.selectedBg` has the value `actionListItem.default.activeBg`

_Note: there was one usage of `btn.selectedBg` going to `actionListItem.default.selectedBg` that I wasn't sure how to remediate_

I think the big thing was just trying to see if we could keep the component tokens (`btn.*`) the same in the source code but change their values in the theme, if possible. Totally get if this really isn't do-able though

Reply:
@joshblack I'm reverting these changes for now and will add them back into the followup PR for our next major release 😄